pkg/service/identity.go

```diff
@@ -32,8 +38,13 @@ func (i *IdentityService) GetPluginCapabilities(context.Context, *csi.GetPluginC
 	}, nil
 }
 
-//Probe
+// Probe checks the state of the connection to ovirt-engine
```

Comment:
what's the frequency of this probe? shouldn't the probe definition is go into the deployment config yaml?